pkg/storage/internal/actions/create.go

```diff
@@ -370,8 +370,8 @@ func (c *create) resultOutputs(newOutputs []*entity.NewOutput) []*wdk.StorageCre
 			},
 		}
 
-		if output.Basket != nil {
-			resultOutputs[i].Basket = to.Ptr(primitives.StringUnder300(*output.Basket))
+		if output.BasketName != nil {
+			resultOutputs[i].Basket = to.Ptr(primitives.StringUnder300(*output.BasketName))
 		}
```

Comment:
I know, that it's not relate to this PR, but hmmm... this should do the similar job 🙈 :
```suggestion
		resultOutputs[i].Basket = (*primitives.StringUnder300)(output.BasketName)
```

And even could be inlined in whole resultOutput[i] creation